README.md

```diff
@@ -177,6 +177,8 @@ Configuration example:
     // 2 - essential game parameters must match (blue P symbol in Pure Faction)
     // 3 - client-side mods are disallowed (gold P symbol in Pure Faction)
     //$DF Anticheat Level: 0
+    // Set click limit for semi-automatic weapons (pistol and precision rifle). Bullets fired quicker than this rate will be ignored. Can be used to lessen impact of autoclicker cheats.
+    $DF Semi Auto Click Limit: 20.0
```

Comment:
maybe "Semi-Auto Click Rate Limit" or "Semi-Auto Fire Rate Limit" would be better?